.github/workflows/ci-release.yml

```diff
@@ -47,10 +47,21 @@ jobs:
           version: latest
           args: check
 
+  # branch_name trims ref/heads/ from github.ref to access a clean branch name
+  branch_name:
+    runs-on: ubuntu-latest
+    outputs:
+      branch: ${{ steps.trim_ref.outputs.branch }}
+    steps:
+      - name: Trim branch name
+        id: trim_ref
+        run: |
+          echo "branch=$(${${{ github.ref }}:11})" >> $GITHUB_OUTPUT
```

Comment:
what is the 11 based on here?

Reply:
My guess was the number of characters in `refs/heads/` but if the string is 0 indexed, then I think 11 is one off so 🤷‍♂️

Reply:
yes, number of characters.